<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskListDialog</class>
 <widget class="QDialog" name="TaskListDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>1102</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>61</width>
    <height>1013</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tasks</string>
  </property>
  <property name="windowIcon">
   <iconset resource="backgroundsrc.qrc">
    <normaloff>:/images/graylogo.png</normaloff>:/images/graylogo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#TaskListDialog{
background: #c3c3c3;
border-image:url(:/images/finalBackground.jpg)
}

QPushButton, QMenuButton, QStatusBar, QTextEdit, QLabel, QListWidget{
font-family: &quot;Roboto Black&quot;;
font-size: 42px;
}


QPushButton{
background-color:white;
}

QPushButton:hover{
background-color: #eb3f79;
color: white;
}




</string>
  </property>
  <widget class="QWidget" name="content" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>950</width>
     <height>706</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>950</width>
     <height>706</height>
    </size>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>951</width>
      <height>701</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>941</width>
      <height>701</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#listWidget{
selection-color: red;
selection-background-color: green;
}
</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="topPanel" native="true">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>0</y>
     <width>1111</width>
     <height>131</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1029</width>
     <height>131</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#topPanel{
border-image:url(:/images/mainHeader.png)
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Title{
font-size: 52px;
font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>List of Tasks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>1020</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeTaskButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>1020</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>61</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>-1</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="input_removeTaskCount">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>940</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>191</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#input_removeTaskCount{
font-size: 42px;
padding-right: 5px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="removeTaskLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>940</y>
     <width>661</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>61</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#Title{
font-size: 52px;
font-weight: bold;
}
#removeTaskLabel{
font-size: 45px;
}</string>
   </property>
   <property name="text">
    <string>Select Task Number To Remove</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="backgroundsrc.qrc"/>
 </resources>
 <connections/>
</ui>
